content/reference/plugins/_index.md

```diff
@@ -6,6 +6,7 @@ ordering:
   - jenkins
   - teamcity
   - bamboo
+  - github_actions
```

Comment:
I added it to the ed of the list of CI plugins... But I just thought that we might want to put the newer CI plugins at the top of the list?